app/controllers/users_controller.rb

```diff
@@ -21,6 +20,7 @@ def create
 
   def show
     @user = current_user
+    @user.rockle = Rockle.find_by({user: User.find(@user.id)})
```

Comment:
Same as above comment. Line isn't necessary. @user.rockle in view would be fine.